packages/types/src/index.ts

```diff
@@ -125,3 +125,7 @@ export type Choice = {
   label?: string
   image?: string
 }
+
+export const EXISTING_BULK_COUPON = 'EXISTING_BULK_COUPON' as const
+export const NEW_BULK_COUPON = 'NEW_BULK_COUPON' as const
+export const NEW_INDIVIDUAL_PURCHASE = 'NEW_INDIVIDUAL_PURCHASE' as const
```

Comment:
These don't really feel to me like they belong here in the `@skillrecordings/types`, but I couldn't find a better place for them. Both `commerce-server` and app client code (e.g. TA Thank You page) use these values.

What I'm imagining is a place where shared constants and Zod schemas can live, where both server and client code and import them. A schema or validation package perhaps, but there isn't enough of an emerging pattern yet to justify a new package.

Any better ideas?

Reply:
I'd probably use a PurchaseType enum for these versus this esoteric const const scheme and don't think the `types` package is an inappropriate spot. If you wanted more specific granularity, a `commerce-types` package would be a great choice too. We don't have to be sparing with the packages. We can eventually create a `commerce` top level workspace/namespace (for instance) if we want to organize/bundle related stuff

Reply:
```
enum PurchaseType {
    EXISTING_BULK_COUPON,
    NEW_BULK_COUPON,
    NEW_INDIVIDUAL_PURCHASE
}
```

results in:

```
var PurchaseType;
(function (PurchaseType) {
    PurchaseType[PurchaseType["EXISTING_BULK_COUPON"] = 0] = "EXISTING_BULK_COUPON";
    PurchaseType[PurchaseType["NEW_BULK_COUPON"] = 1] = "NEW_BULK_COUPON";
    PurchaseType[PurchaseType["NEW_INDIVIDUAL_PURCHASE"] = 2] = "NEW_INDIVIDUAL_PURCHASE";
})(PurchaseType || (PurchaseType = {}));
```

and is WAY easier for me to read than the `const const zod` and the enum is simple to use as a type